plotmanager/library/utilities/jobs.py

```diff
@@ -50,7 +50,8 @@ def check_valid_destinations(job, drives_free_space):
 
     valid_destinations = []
     for directory in destination_directories:
-        drive = identify_drive(file_path=directory, drives=drives)
+        #drive = identify_drive(file_path=directory, drives=drives)
```

Comment:
this seems wrong. did you test? what if destination is /mnt/drive/subfolder (or X:\subfolder)? 
even if this is correct, directory is not a drive. you should change names to not confuse other developers

Reply:
The identify_drive function was not working correctly.  Since we already input the correct target dir, we do not need check it anymore. 



<img width="486" alt="x" src="https://user-images.githubusercontent.com/9717716/121009149-695bad80-c7c6-11eb-97e1-396d3cc8f5f4.png">